src/LibraryManagement.Application/CommandHandlers/Authentication/CreateUserCommandHandler.cs

```diff
@@ -35,7 +35,7 @@ public async Task<Guid> Handle(CreateUserCommand command, CancellationToken canc
         var result = await _userManager.CreateAsync(user, command.Password);
 
         if (!result.Succeeded)
-            throw new ValidationException("Invalid information");
+            throw new ValidationException(result.Errors.FirstOrDefault()?.Description);
```

Comment:
In case, you have more than two errors, how will you handle it?

Reply:
Will handle this case in next PR